Kiosk/Views/InkRecognizerExplorer/InkRecognizerExplorer.xaml.cs

```diff
@@ -0,0 +1,69 @@
+﻿// 
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// 
+// Microsoft Cognitive Services: 
+// http://www.microsoft.com/cognitive
+// 
+// Microsoft Cognitive Services Github:
+// https://github.com/Microsoft/Cognitive
+// 
+// Copyright (c) Microsoft Corporation
+// All rights reserved.
+// 
+// MIT License:
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+// 
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED ""AS IS"", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+// 
+
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace IntelligentKioskSample.Views.InkRecognizerExplorer
+{
+    [KioskExperience(Title = "Ink Recognizer Explorer", ImagePath = "ms-appx:/Assets/InkRecognizerExplorer.png")]
+    public sealed partial class InkRecognizerExplorer : Page
```

Comment:
Typically in the kiosk demos we have an override for the OnNavigatedTo method to check for the Api Keys. It then either continues with the loading, or shows a message and blocks the relevant aspects of the UI so the user can't make requests. See an example [here](https://github.com/microsoft/Cognitive-Samples-IntelligentKiosk/blob/master/Kiosk/Views/TranslatorExplorer/TranslatorExplorerPage.xaml.cs#L152).

Please do that here as well.

Reply:
The message displayed for missing an API key is now only displayed once when entering the Ink Recognizer Explorer demo and if the key is missing the UI is disabled